### PR TITLE
P0 and q0 of batteries

### PR DESCRIPTION
**Pull request by anas-drissi**

### CODE REVIEW

File: src/components/network/config-tables.js [hunk 1]
```diff
@@ -918,13 +918,13 @@ export const TABLES_DEFINITIONS = {
                 canBeInvalidated: true,
             },
             {
-                id: 'ConstantActivePower',
+                id: 'ConsigneActivePower',
```
Comment (sBouzols) [inline src/components/network/config-tables.js:921]: use 'TargetP' and 'TargetQ' existing translation keys

File: src/translations/en.json [hunk 2]
```diff
@@ -236,8 +236,10 @@
     "LccConverterStations": "LCC Converter Stations",
     "VscConverterStations": "VSC Converter Stations",
     "LoadType": "Type",
-    "ConstantActivePower": "Target p (MW)",
-    "ConstantReactivePower": "Target q (MVar)",
+    "ConsigneActivePower": "Target p (MW)",
+    "ConsigneReactivePower": "Target q (MVar)",
+    "ConstantActivePower": "Constant p (MW)",
```
Comment (sBouzols) [inline src/translations/en.json:241]: Rename Keys 'ConstantActivePower' -> 'ConstantP'
'and ConstantReactivePower' -> 'ConstantQ'

Comment (anas-drissi) [reply src/translations/en.json]: it's fixed.

File: src/components/network/config-tables.js (hunk 1 above)
Comment (anas-drissi) [reply src/components/network/config-tables.js]: it's fixed.

**Review by sBouzols — APPROVED** [review]

**Pull request merged**